MAUI/DataForm/Validation.md

```diff
@@ -49,6 +49,97 @@ public DateTime JoinDate { get; set; }
 
 Refer [here](https://help.syncfusion.com/maui/dataform/data-annotations#dateformdaterange-attribute) to know more about date range attribute in DataForm.
 
+#### IDataErrorInfo
+
+You can validate the data by implementing the [IDataErrorInfo](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.idataerrorinfo?view=net-6.0) interface in the data object class.
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : IDataErrorInfo
+{
+    public string EmployeeID { get; set; }
+
+    public string Name { get; set; }
+
+    public string Title { get; set; }
+
+    [Display(AutoGenerateField = false)]
+    public string Error
+    {
+        get
+        {
+            return string.Empty;
+        }
+    }
+
+    [Display(AutoGenerateField = false)]
+    public string this[string columnName]
+    {
+        get
+        {
+            if (columnName == nameof(Title) && this.Title.Contains("Marketing"))
+            {
+                return "Marketing is not allowed";
+            }
```

Comment:
Change 3#
### Validate using the validation attribute
### Validate using the IDataErrorInfo interface
### Validate using the INotifyDataErrorInfo inferface

Reply:
changed